### PR TITLE
Itm 589

### PR DESCRIPTION
**Pull request by dereknop**
[Dashboard pr](https://github.com/NextCenturyCorporation/itm-evaluation-dashboard/pull/117)

From venv run `python3 deployment_script.py`

The original json configs from MRE should now be added to the textBasedConfig Mongo collection

### CODE REVIEW

File: text_based_scenarios/_0_2_5_add_mre_json.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    
+    current_script_dir = os.path.dirname(os.path.abspath(__file__))
+    json_folder = os.path.join(current_script_dir, 'mre-json-files')
+    
+    # Get all JSON files in the folder
+    json_files = [f for f in os.listdir(json_folder) if f.endswith('.json')]
+    
+    for file in json_files:
+        file_path = os.path.join(json_folder, file)
+        with open(file_path, 'r') as f:
```
Comment (nextcen-dgemoets) [inline text_based_scenarios/_0_2_5_add_mre_json.py:15]: The python `open` call always needs an extra parameter for me, `encoding='utf-8'`.  I was going to say that Windows needs it, but apparently you don't need it...

**Review by nextcen-dgemoets — COMMENTED** [review]
I'm seeing the MRE results in the `textBasedConfig` table, but not in the dashboard in the `ITM-589` branch.  Is there some other prerequisite PR I need to have done?

[Edit:  I ran dashboard PR `ITM-589` as well and ran `bash startup-scripts/dev_graphql_dashboard_ui.sh` afterwards.]

**Review by kaitlyn-sharo — APPROVED** [review]
Worked as expected. Had to make sure I had run _0_1_9 first to add evalIDs to text based data. I guess I missed that one. Derek has made a ticket so that from now on, the deployment script will run all of the scripts you have missed between your current db version and the new one (i.e. if you're in version 1.0 and db is 2.0, it'll not just run the most recent script, but every script 1.1, 1.2, 1.3, etc)

**Comment by nextcen-dgemoets** [comment]
The ingest looks good.  I have 27 entries in my `textBasedConfig` Mongo collection.

**Review by nextcen-dgemoets — APPROVED** [review]

**Review by brianpippin — APPROVED** [review]

**Pull request merged**